inbm/dispatcher-agent/tests/unit/schedule/test_manifest_parser.py

```diff
@@ -6,10 +6,12 @@
 
 GOOD_IMMEDIATE_SCHEDULE_XML = """<?xml version="1" encoding="utf-8"?>
 <schedule_request>
-    <request_id>REQ12345</request_id>
+    <request_id>4324a262-b7d1-46a7-b8cc-84d934c3983f</request_id>
     <update_schedule>
         <schedule>
-            <single_schedule />
+            <single_schedule>
+                <job_id>swupd-939fe48c-32da-40eb-a00f-acfdb43a5d6d</job_id>
+            </single_schedule>
         </schedule>
         <manifests>
             <manifest_xml><![CDATA[<?xml version="1.0" encoding="utf-8"?><manifest><type>ota</type><ota><header><type>sota</type><repo>remote</repo></header><type><sota><cmd logtofile="y">update</cmd><mode>full</mode><deviceReboot>no</deviceReboot>
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** Ensure XML strings are properly formatted and adhere to the expected schema for schedule requests.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
GOOD_IMMEDIATE_SCHEDULE_XML = """<?xml version="1.0" encoding="UTF-8"?>
<schedule_request>
    <request_id>4324a262-b7d1-46a7-b8cc-84d934c3983f</request_id>
    <update_schedule>
        <schedule>
            <single_schedule>
                <job_id>swupd-939fe48c-32da-40eb-a00f-acfdb43a5d6d</job_id>
            </single_schedule>
        </schedule>
        <manifests>
            <manifest_xml><![CDATA[<?xml version="1.0" encoding="UTF-8"?><manifest><type>ota</type><ota><header><type>sota</type><repo>remote</repo></header><type><sota><cmd logtofile="y">update</cmd><mode>full</mode><deviceReboot>no</deviceReboot></sota></type></ota></manifest>]]></manifest_xml>
        </manifests>
    </update_schedule>
</schedule_request>"""
```